### PR TITLE
Varnish - Add checksum of configmap to the pods

### PR DESCRIPTION
**Pull request by kierenevans**
To force re-deploy when the configmap changes

### CODE REVIEW

File: src/_base/helm/app/templates/service/varnish/statefulset.yaml [hunk 1]
```diff
@@ -15,6 +15,8 @@ spec:
   template:
     metadata:
       creationTimestamp: null
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/service/varnish/configmap.yaml") . | sha256sum }}
```
Comment (andytson-inviqa) [inline src/_base/helm/app/templates/service/varnish/statefulset.yaml:19]: some have experienced the sha256 is too long for an annotation, but hopefully that is just in a k8s version lower than EKS 1.17

Comment (kierenevans) [reply src/_base/helm/app/templates/service/varnish/statefulset.yaml:19]: Could do `sha1sum`?

Comment (andytson-inviqa) [reply src/_base/helm/app/templates/service/varnish/statefulset.yaml:19]: seems to have changed 4 years ago to 256 characters. sha256 is 64 (where limit seemed to trigger as if annotation was 65)

Comment (andytson-inviqa) [reply src/_base/helm/app/templates/service/varnish/statefulset.yaml:19]: k8s 1.1 was the version that size limit was released in, so I think we're safe

**Review by andytson-inviqa — APPROVED** [review]

**Pull request merged**